comixed-frontend/src/app/library/library.constants.ts

```diff
@@ -16,9 +16,9 @@
  * along with this program. If not, see <http://www.gnu.org/licenses>
  */
 
-import { API_ROOT_URL } from 'app/app.functions';
-import { COMIXED_API_ROOT } from 'app/app.constants';
-import { ReadingList } from 'app/comics/models/reading-list';
+import {API_ROOT_URL} from 'app/app.functions';
+import {COMIXED_API_ROOT} from 'app/app.constants';
+import {ReadingList} from 'app/comics/models/reading-list';
```

Comment:
I thought prettier was adding a space after the opening brace and the ending brace. Could you check this ?

Reply:
Correct, I'll go back and fix this and repush.